cmake/modules/extensions.cmake

```diff
@@ -2996,6 +2996,31 @@ function(target_byproducts)
   )
 endfunction()
 
+# Usage:
+#   get_current_scope(<out-variable> <variable>)
+#
+# This function reliably obtains the value of <variable> in the current scope
+# and returns it in <out-variable>. If <variable> is not defined in that scope,
+# then <out-variable> becomes unset.
+#
```

Comment:
I don't believe this matches what the code is doing, on line 3017 it is not setting the value of the provided variable, not setting the variable itself?

Reply:
Seems to be OK